ruby/2.7.5/Dockerfile

```diff
@@ -41,9 +42,13 @@ RUN yum install -y \
         redis \
         sqlite-devel \
         readline-devel
-
-RUN yum install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm
-RUN yum install -y postgresql-server12 postgresql-contrib12 postgresql-devel12
+COPY VA-Internal-S2-RCA1-v1.cer /etc/pki/ca-trust/source/anchors/VA-Internal-S2-RCA1-v1.cer
```

Comment:
i think a note should be added here that this assumes this Dockerfile is being ran in the VAEC...

Reply:
from lines 45 - 47 should be noted

Reply:
Okay, I added a comment. Thanks Aurora! 😄

Reply:
excellent! LGTM!!!